apps/src/gamelab/animationListModule.js

```diff
@@ -291,6 +335,20 @@ export function addBlankAnimation() {
   };
 }
 
+export function addBlankAnimationAddition() {
+  return (dispatch, getState) => {
+    var selectedAnimationKey = getState().animationTab.selectedAnimation;
+    dispatch({
+      type: SET_PENDING_ANIMATION_ADDITION,
+      key: selectedAnimationKey,
+      props: {
+        blankFrame: true
+      }
+    });
```

Comment:
Here and elsewhere:

Not sure if Gamelab has been doing differently, but elsewhere we've tried to have individual action creators for each action, so we'd have something like:
```
dispatch(setPendingAnimationAddition(selectedAnimationKey))
```
and elsewhere
```
function (key) {
  return {
    type: SET_PENDING_ANIMATION,
    key,
    props: {
      blankFrame: true
    }
}
```